example/horizontallist.gui

```diff
@@ -66,6 +66,7 @@ nodes {
   alpha: 1.0
   template_node_child: false
   size_mode: SIZE_MODE_MANUAL
+  custom_type: 0
```

Comment:
*thought*: I don't know what these are about, but the editor added it to all components.